src/alitra/models/orientation.py

```diff
@@ -0,0 +1,67 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+import numpy as np
+from scipy.spatial.transform import Rotation
+
+from .frame import Frame
+
+
+@dataclass
+class Orientation:
+    """
+    This class represents an orientation scipy rotation object and a frame
+    """
+
+    x: float
+    y: float
+    z: float
+    w: float
```

Comment:
I prefer the previous solution where the dataclass holds a rotation object, instead of the four quaternion variables